### PR TITLE
Create locales for Afaan Oromoo and Amharic

### LINKED ISSUE #2179: Create moment locale for - Afaan Oromoo, Amharic
**Is your feature request related to a problem? Please describe.**
Moment doesn't have a locale for Afaan Orommo and Amharic.

**Describe the solution you'd like**
Create a locale for Afaan Orommo(om) and Amharic (am) in `psammead-locales`. 
Get the locales [here](https://repl.it/repls/ComplicatedCluelessTab)

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
Resolves #2179

**Overall change:**  Add moment locales for Afaan Oromoo and Amharic.
**Code changes:**

-  Add locale files with months and relative time
- Add unit tests.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/moment/index.stories.jsx [hunk 1]
```diff
@@ -50,6 +52,8 @@ const locales = [
   { name: 'Ukrainian', locale: 'uk' },
   { name: 'Yoruba', locale: 'yo' },
   { name: 'Kyrgyz', locale: 'ky' },
+  { name: 'Afaan Oromoo', locale: 'om' },
+  { name: 'Amharic', locale: 'am' },
```
Comment (DenisHdz) [inline packages/utilities/psammead-locales/moment/index.stories.jsx]: LGTM, but could you sort the locales here by name?

Comment (ghost) [reply packages/utilities/psammead-locales/moment/index.stories.jsx]: Sorted.

**Review by DenisHdz — APPROVED** [review]

**Review by AlistairGempf — APPROVED** [review]
Looks good to me.

**Review by j-pendlebury — APPROVED** [review]

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**